core/src/main/java/io/questdb/griffin/engine/table/ShowPartitionsRecordCursorFactory.java

```diff
@@ -377,7 +377,11 @@ public long getLong(int col) {
                 if (Column.DISK_SIZE.is(col)) {
                     return partitionSize;
                 }
-                throw new UnsupportedOperationException();
+                if (Column.PARTITION_INDEX.is(col)) {
+                    return dynamicPartitionIndex;
```

Comment:
there are two timestamps:

![image](https://user-images.githubusercontent.com/7276403/233136697-f3172285-3a03-44e9-bef9-1e5ba970dadc.png)

Reply:
```java 
return getTimestamp(col) 
``` 
below should deal with them.